### DISCUSSION TITLE
醫療工作站USM-501🖥

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#87] [General]
**新聞大綱：**
研華日前推出醫療級 AI 工作站「USM-501」，為**醫療 3D 渲染、影像優化及影像診斷解析的運算載體**，藉由高速資訊運算及分析能力、精準成像醫療儀器畫面，協助醫生精準判斷，醫院更可藉此逐年檢視並更新院內醫療設備，逐步建置智慧醫院建設並進行數位化轉型。

另外，USM-501 不僅具有軟硬體整合優勢外，更滿足醫療院**品牌客製化**需求。客戶或院方系統開發人員能以 USM-501 為基礎快速創造獨特方案，縮短整體上線時程。同時是全球首款獲 NVIDIA 認證系統之認證產品，將為醫院手術設備、醫療診斷儀器及實驗室環境提供搭載邊緣人工智慧技術的平台。

此次 NVIDIA以及研華不僅研發出最適醫療客戶的產品組合，簡化終端應用印證時間與流程；更藉由 NVIDIA 認證系統認證，確保醫療數據安全性、設備可管理性、擴充性與優化硬體效能等功能。

**新聞心得：**
在這則新聞中，感覺這項攜手合作的機器結合了許多迅速、方便、準確的功能，或許未來許多醫院採用此設備後，能讓更多醫院有高科技設備輔助分析、判斷，可以縮短漫長的作業時間、優化整體流程，但是從外觀看起來，不像是一個會做這些事情的機器！

**新聞來源：**
https://technews.tw/2021/10/28/usm-501/

![圖說：醫療級AI人工智慧工作站-USM-501，為全球首款獲-NVIDIA認證系統™之認證產品，將藉由高速資訊運算及分析能力，快速呈現清晰與精密影像畫面，協助醫生精準判斷病灶並提升病](https://user-images.githubusercontent.com/90883526/139210404-44f1f7e7-3261-474e-b445-d3380a6a5a43.jpg)

**Comment by boson13579** [comment]
從外觀看起來就是...我很會算喔~沒了，ㄏㄏ
AI 導入醫療是未來的趨勢拉，硬體肯定也是要跟上的，不然開個刀，都在等電腦算也太蠢XD
